### PR TITLE
chore: tool file corrected

### PR DESCRIPTION
**Pull request by akshatnema**
**Description**
The `.asyncapi-tool` file is corrected according to the errors made while validating with the [schema](https://github.com/asyncapi/website/blob/master/scripts/tools/tools-schema.json).

### CODE REVIEW

File: .asyncapi-tool [hunk 1]
```diff
@@ -1,6 +1,8 @@
+---
 title: GitHub Action for Generator
 filters:
-    technology:
-        - AsyncAPI Generator
-    categories:
-        - github-actions
+  language: JavaScript
```
Comment (derberg) [inline .asyncapi-tool:4]: no language, it is not a library, it is GH action, nobody can integrate it in their code

File: .asyncapi-tool [hunk 2]
```diff
@@ -1,6 +1,8 @@
+---
```
Comment (derberg) [inline .asyncapi-tool:1]: this is not needed, right?

**Comment by derberg** [comment]
@akshatnema ping

**Comment by akshatnema** [comment]
pinging @derberg :eyes:

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
> pinging @derberg 👀

@akshatnema you don't know the game, you should reply with `pong` 😛 

/rtm

**Pull request merged**